router/router.go

```diff
@@ -122,6 +131,8 @@ type HandleT struct {
 	savePayloadOnError                     bool
 	oauth                                  oauth.Authorizer
 	transformerProxy                       bool
+	allowRtAbortAlertForDelv               bool // represents if transformation(router or batch) should be alerted via router-aborted-count alert def
+	allowRtAbortForTf                      bool // represents if event delivery(via transformerProxy) should be alerted via router-aborted-count alert def
```

Comment:
I am a bit confused with the comments of these fields and how they are actually being used in `getAllowedRtTags`.

E.g. it seems to me that their names should be:
`allowRtAbortForTf` --> `alertOnTransformationAbort // send an alert if an event is aborted due to a transformation error`
`allowRtAbortAlertForDelv` --> `alertOnTransformerProxyAbort  // send an alert if an event is aborted by transformer proxy`

Reply:
The detail of name(s) chosen
`allowRtAbortForTf` - allow router-aborted-count alert for transformation
`allowRtAbortForDelv` - allow router-aborted-count alert for event Delivery
These are specific to controlling alert through `router-aborted-count` alert definition

**Note**:
- `Abort` in the names chosen doesn't reflect the (aborted)status of the job/event


Let me know if it makes sense